src/main/java/org/triplea/client/ui/javafx/ClientSettingJavaFxUiBinding.java

```diff
@@ -0,0 +1,151 @@
+package org.triplea.client.ui.javafx;
+
+import java.util.Map;
+import java.util.function.Supplier;
+
+import com.google.common.base.Suppliers;
+
+import games.strategy.triplea.settings.ClientSetting;
+import games.strategy.triplea.settings.GameSetting;
+import games.strategy.triplea.settings.GameSettingUiBinding;
+import games.strategy.triplea.settings.SelectionComponent;
+import games.strategy.triplea.settings.SettingType;
+import javafx.scene.layout.Region;
+
+enum ClientSettingJavaFxUiBinding implements GameSettingUiBinding<Region> {
+  AI_PAUSE_DURATION_BINDING(
+      SettingType.AI,
+      JavaFxSelectionComponentFactory.intValueRange(ClientSetting.AI_PAUSE_DURATION, 0, 3000)),
```

Comment:
Maybe not as part of this PR, but should we consider extracting these constraints/limits somehow so that they can be shared between the Swing and JFX UIs?

Reply:
Perhaps.
Currently this has the benefit that we can make the individual screens leave stuff out, but we can probably reproduce this by using optionals or something

Reply:
Sorry for not clarifying...  I was talking specifically about, in this particular case, the fact that the AI pause duration has a legal range between 0 and 3,000 ms.  That's purely a domain model constraint and should be independent of the UI.

The most straightforward (but verbose) solution would be to define constants for these values that can be shared between `ClientSettingSwingUiBinding` and `ClientSettingJavaFxUiBinding`.  A more complex solution would be to define some kind of polymorphic constraint type and attach it directly to `GameSetting`.

Reply:
Option 2 sounds better to me.
I'd suggest doing this in a seperate PR though to keep the diff smaller.
Any specific suggestions?

Reply:
Agreed, a separate PR is best.  No specific suggestions at this time.